README.md

```diff
@@ -329,6 +329,13 @@ Once installed, you can use the following configuration data:
 extension="/path/to/tideways/tideways_xhprof.so"
 ```
 
+Monitoring
+==========
+
+[Prometheus](https://prometheus.io) metrics suitable for monitoring service
+health are exposed on `/metrics`.  (This currently only works if using PDO for
+storage.)
+
```

Comment:
perhaps use `/export/prometheus` path to be similar to some (not yet merged) exporter:
- https://github.com/perftools/xhgui/pull/302/files

Reply:
In fairness, those exporters tend to have a dedicated HTTP host or port just for their Prometheus exporter. I haven't checked them all, but I don't see much if any precedent for how to expose Prometheus from something that is itself a CGI web application.

I suppose there's a number of ways forward here:

1. Integrate the endpoint within the regular XHGui REST paths as `/export/prometheus` and use declare the custom path in Prometheus (is this possible?)
2. Integrate as `/export/prometheus` and document for Nginx and Apache how to configure a second virtualhost or port that exposes `/metrics` proxy-path'ed to this endpoint. This would be most complient with [Prometheus best practices](https://prometheus.io/docs/instrumenting/writing_exporters/) in that it would allow the XHGui exporter to have a unique port in the 9xxx range and avoids clashes with any other path conventions it might introduce later. For example, it recommends that visiting http://myexporter:9xxx/ shows a page that links to `/metrics`.
3. Integrate as `/metrics` and treat the public XHGui app itself as a Prometheus exporter.

Which one of these to follow I'll leave up to @glensc.

Reply:
I have no strong feelings about the URL path, so be it `/metrics` for now.